### ISSUE TITLE
Menu: (Accessibility)

### ISSUE THREAD

**Issue by kodecount** [wet-boew/wet-boew#7795]
What is the purpose of `aria-posinset="1" aria-setsize="1"` and are they breaking any rule by being attached to a link element.

According to AXE they could break this rule: https://dequeuniversity.com/rules/axe/2.1/aria-allowed-attr?application=axeAPI

**Comment by duboisp** [comment]
> Some states are managed by the user agent, such as ```aria-posinset``` and ```aria-setsize```, but the author can override them if the DOM is incomplete and would cause the user agent calculation to be incorrect. - [2.2 WAI-ARIA States and Properties](https://www.w3.org/TR/wai-aria-1.1/#introstates)

FYI - here the spec of :
* [aria-posinset](https://www.w3.org/TR/wai-aria-1.1/#aria-posinset)
* [aria-setsize](https://www.w3.org/TR/wai-aria-1.1/#aria-setsize)

@kodecount it is about a specific use case? if so can you provide more details.

**Comment by kodecount** [comment]
@duboisp Thanks for the information. I'm using JAWs, WAVE Evaluation Tool, and aXe in Chrome for accessibility testing one of the applications had a issue with the aria controls. I believe it due to the fact that their intended for list elements and not link elements except provide the same purpose in this instance.

**Comment by kodecount** [comment]
@duboisp Thanks for the information. I'm using JAWs, WAVE Evaluation Tool, and aXe in Chrome for accessibility testing one of the applications had a issue with the aria controls.

I think I see why it has a issue with the aria controls. Maybe these should be applied to the `<li>` tag and not the `<a>`? Is their a reason you choice to add them to the link versus the list item?

**Comment by kodecount** [comment]
@duboisp Thanks for the information. I'm using JAWs, WAVE Evaluation Tool, and aXe in Chrome for accessibility testing one of the applications had a issue with the aria controls.

I think I see why it has a issue with the aria controls. Maybe these should be applied to the `<li>` tag and not the `<a>`? Is their a reason WET choose to add them to the link versus the list item?
